Whack-a-mole/main.js

```diff
@@ -0,0 +1,81 @@
+const holes = document.querySelectorAll(".hole");
+const scoreBoard = document.querySelector(".score");
+const moles = document.querySelectorAll(".mole");
+const level1 = document.querySelector(".easy");
+const level2 = document.querySelector(".medium");
+const level3 = document.querySelector(".hard");
+const record = document.querySelector(".record");
+let lastHole;
+let timeUp = false;
+let score = 0;
+let recording = localStorage.getItem("totalScore");
```

Comment:
you need a value for initial loading when locaStorage is empty

Reply:
done

Reply:
typically use such expression `let recording = localStorage.getItem("totalScore") || 0;`